### ISSUE TITLE
DreamBoth LoRA regularization

### ISSUE THREAD

**Issue by weiwei83** [bmaltais/kohya_ss#1121]
Because the picture background and lighting in the data set are relatively simple, I choose to use regularization to train the data set.
But I found that after using regularization, my training results did not achieve the effect.
Below are my training parameters and training results.

 accelerate launch 
--num_cpu_threads_per_process=2 "train_network.py" 
--pretrained_model_name_or_path="my_model_path/chilloutmix_NiPrunedFp32Fix.safetensors"                         
--train_data_dir="D:/train/YeonHwa/images 
--reg_data_dir="D:/train/YeonHwa/reg" 
--resolution="512,768"    
--network_dim=128                     
--network_alpha="32"                         
 --network_module=networks.lora 
--text_encoder_lr=5e-06 
--unet_lr=2e-05                      
--learning_rate="2e-05" 
--lr_scheduler_num_cycles="20" 
--optimizer_type="AdamW8bit"                         
--lr_scheduler="constant"                         
--train_batch_size="3" 
--max_train_steps="2667" 
--save_every_n_epochs="2" 
--mixed_precision="bf16" 
--save_precision="bf16"                         
--cache_latents 
--keep_tokens="2" 
--bucket_reso_steps=64 
--shuffle_caption 
--xformers 
--bucket_no_upscale 
--noise_offset=0.06        
……              

My training set has 40 pictures, including various postures of the face, bust, full body, etc. The number of repeats is 5 and fixed, and only the number of regular pictures changes.
The regular image is generated using the keyword "woman".
The training results from left to right in the figure below are: original model, no regular pictures, 100 regular pictures, 10 regular pictures, 200 regular pictures, 50 regular pictures
<img width="1267" alt="image" src="https://github.com/bmaltais/kohya_ss/assets/10830200/8c6e4480-db4a-47f6-8e51-2719aa661386">

Can someone tell me what I'm doing wrong, and how I need to modify it to maintain the characteristics of the model while increasing its generalization?

**Comment by aearone** [comment]
Use more training steps and also increase Network Dim and Network Alpha to 128 and 128 (or half of Network Dim). Aslo you can give more creativity to the neural network by decreasing the Alpha value but increase the number of training steps to prevent undertraining. This is where you need to test. Did you use BLIP captions for each of the dataset photos?
